public/components/apps/app.jsx

```diff
@@ -45,9 +56,9 @@ export default class App extends React.Component {
     })
   }
 
+ //I really don't know what is the function of this if there is another one on explorable-app 
```

Comment:
I think it is a legacy function that does not work anymore. But cannot remember. The one that should be running must be on `explorable-app.jsx`, therefore this function (`App.launch`) can be removed.